doc/sphinx/Makefile

```diff
@@ -9,11 +9,33 @@ SPHINXBUILD   ?= sphinx-build
 SOURCEDIR     = .
 BUILDDIR      = _build
 VENVDIR       = $(SPHINXDIR)/venv
-PA11Y         = $(SPHINXDIR)/node_modules/pa11y/bin/pa11y.js --config $(SPHINXDIR)/pa11y.json
+PA11YCI       = $(SPHINXDIR)/node_modules/pa11y-ci/bin/pa11y-ci.js --config $(SPHINXDIR)/pa11y-ci.json
 VENV          = $(VENVDIR)/bin/activate
-
-.PHONY: help full-help woke-install pa11y-install install run html epub serve \
-        clean clean-doc spelling linkcheck woke pa11y Makefile
+WORKDIR	      = ../build_doc
+WORKSPHINX    = $(WORKDIR)/doc/sphinx
+PROJROOT      = ../..
+DEPS	      = cmake doxygen libboost-dev libboost-iostreams-dev \
+		libboost-filesystem-dev libboost-program-options-dev \
+		libboost-system-dev libdrm-dev libegl-dev libepoxy-dev \
+		libfreetype-dev libglib2.0-dev libgles2-mesa-dev \
+		libglm-dev libinput-dev liblttng-ust-dev libwayland-dev \
+		libxcb-composite0-dev libxcb-randr0-dev libxcursor-dev \
+		libxkbcommon-dev libxml++2.6-dev libyaml-cpp-dev xsltproc
+
+# If the environment variable `GITHUB_WORKFLOW` is empty, then we're running
+# locally and shouldn't depend on `_build/index.html` to build HTML (so even if
+# it exists, we'll rebuild)
+# On CI where the envvar is set, we should depend on `_build/index.html` so we
+# don't build multiple times (multiple invocations = multiple complete builds
+# even if files didn't change)
+ifeq ($(GITHUB_WORKFLOW), "")
+HTMLINDEX     = ""
+else
+HTMLINDEX     = $(WORKSPHINX)/_build/index.html
+endif
+
+.PHONY: help full-help woke-install pa11y-ci-install install run html epub serve \
+        clean clean-doc spelling linkcheck woke pa11y-ci Makefile
```

Comment:
```suggestion
```

Reply:
Why remove the phony target?

Reply:
Because if you don't add `-ci` to `pa11y` above, it's there already

Reply:
Should be handled now